python/ray/tests/test_channel.py

```diff
@@ -140,11 +140,137 @@ def _test(val):
     _test(1000)
     _test(np.random.rand(10))
 
-    # Cannot put a serialized value larger than the allocated buffer.
-    with pytest.raises(ValueError):
-        _test(np.random.rand(100))
 
-    _test(np.random.rand(1))
+def test_multiple_channels_different_nodes(ray_start_cluster):
```

Comment:
Can you add a docstring to explain what the test does?

Reply:
Done. Do you want me to add docstrings for all of the tests?

Reply:
That would be great! I don't always do it if the test name is straightforward enough, but it's a good practice.

Reply:
Also, typically we put the docstring on the line after `def` for python.